src/index.html

```diff
@@ -7,6 +7,56 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section>
+      <div class="calendar">
+        <div class="start-day">
+          <div class="start-day__day start-day__day--mon"></div>
+          <div class="start-day__day start-day__day--tue"></div>
+          <div class="start-day__day start-day__day--wed"></div>
+          <div class="start-day__day start-day__day--thu"></div>
+          <div class="start-day__day start-day__day--fri"></div>
+          <div class="start-day__day start-day__day--sat"></div>
+          <div class="start-day__day start-day__day--sun"></div>
+        </div>
+
+        <div class="calendar__block">
+          <div class="calendar__cell calendar__cell--1"></div>
```

Comment:
```suggestion
          <div class="calendar__cell"></div>
```
don't need to set it for all days

Reply:
Чому?? Якщо просто вішати before на calendar__cell то воно ж в циклі просто перебиває і оставляє останнє значення...

Reply:
перебиває що? 
просто циклом через `nth-child` додавай потрібні стилі